### DISCUSSION TITLE
How to scroll down the measurement graph

### DISCUSSION THREAD

**Discussion by RicardoKers** [hneemann/Digital#849] [Q&A] [answered]
I have an measurement graph with many measurements and I can't scroll down to see all the signals. The zoom buttons didn't work either. See an image attached.
is there any way to scroll the window down?
I'm using version 0.28 in Portuguese on windows 11 64 bits, and java is 1.8.0_30

Thanks for the help and congratulations for the great work.

![digital 1](https://user-images.githubusercontent.com/49044489/135247487-881b908c-5ae2-4cd8-bd92-c282d1dd6248.png)

**Comment by hneemann** [comment, marked as answer]
This is not possible! It almost looks like I overlooked this detail when implementing it. :smile:

**Comment by RicardoKers** [comment]
Got it, thanks for the reply.

**Reply by hneemann** [reply]
@RicardoKers 
Please try the latest [pre release](https://infdigital.dhbw-mosbach.de/) and let me know how it works.

**Comment by RicardoKers** [comment]
I tested the pre release and the scroll down of the measurement graph worked perfectly. Thank you for your efforts in solving the problem.

![Digital2](https://user-images.githubusercontent.com/49044489/135446447-902b803b-f5ae-4db5-b7c8-dc050acb25c1.png)

**Reply by hneemann** [reply]
You are welcome!
